drink/src/mock/mocking_api.rs

```diff
@@ -1,12 +1,11 @@
 use crate::{
-    contract_api::ContractApi,
     mock::ContractMock,
     runtime::{AccountIdFor, Runtime},
     Sandbox, DEFAULT_GAS_LIMIT,
 };
 
 /// Interface for basic mocking operations.
-pub trait MockingApi<R: Runtime> {
+pub trait MockingApi<R: pallet_contracts::Config> {
```

Comment:
What's the reason for removing `ChainApi` and `ContractApi` traits but leaving this one?